alembic/env.py

```diff
@@ -8,7 +8,31 @@
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
-config.set_main_option("sqlalchemy.url", os.environ["RSTUF_SQL_SERVER"])
+
+# Support RSTUF Worker Container multiple environment variables for SQL
+sql_server = os.getenv("RSTUF_SQL_SERVER")
+sql_user = os.getenv("RSTUF_SQL_USER")
+sql_password = os.getenv("RSTUF_SQL_PASSWORD")
+
+if sql_server is None:
+    raise ValueError("RSTUF_SQL_SERVER is required")
+
+sql_server = sql_server.replace("postgresql://", "")  # remove protocol
+
+if sql_user and sql_password is None:
```

Comment:
I think you also need to add a case when `sql_user is None and sql_password`.

Reply:
The Postgresql client handles this case in Alembic.

If we handle it, the user loses the possibility to use PostgreSQL [without a password (trust mode)](https://hub.docker.com/_/postgres)(see `POSTGRES_HOST_AUTH_METHOD`)

This is handled as a no password supplied
```log
repository-service-tuf-worker_1  |     return self.loaded_dbapi.connect(*cargs, **cparams)
repository-service-tuf-worker_1  |   File "/usr/local/lib/python3.10/site-packages/psycopg2/__init__.py", line 122, in connect
repository-service-tuf-worker_1  |     conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
repository-service-tuf-worker_1  | sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) fe_sendauth: no password supplied
repository-service-tuf-worker_1  | 
repository-service-tuf-worker_1  | (Background on this error at: https://sqlalche.me/e/20/e3q8)
repository-service-tuf-worker_1  | Failed to initiate the database
```